R/importWQP.R

```diff
@@ -90,15 +90,20 @@ importWQP <- function(obs_url, zip=FALSE, tz=""){
       headerInfo <- attr(doc, "headerInfo")
     }
     
-    suppressWarnings(namesData <- read.delim(if(zip) doc else textConnection(doc) , header = TRUE, quote="",
-                                             dec=".", sep='\t', colClasses='character',nrow=1))
-    
-    classColumns <- setNames(rep('character',ncol(namesData)),names(namesData))
-    
-    classColumns[grep("MeasureValue",names(classColumns))] <- NA
+    library(readr)
```

Comment:
I'll be updating this, but put this in the roxygen next time for it to get to the NAMESPACE